protob/messages-monero.proto

```diff
@@ -137,11 +131,8 @@ message MoneroTransactionInitRequest {
         optional uint64 fee = 8;
         optional uint32 account = 9;
         repeated uint32 minor_indices = 10;
-        optional bool is_multisig = 11;
-        optional bytes exp_tx_prefix_hash = 12;
-        repeated bytes use_tx_keys = 13;
-        optional MoneroTransactionRsigData rsig_data = 14;
-        repeated uint32 integrated_indices = 15;
+        optional MoneroTransactionRsigData rsig_data = 11;
+        repeated uint32 integrated_indices = 12;
```

Comment:
User can enter payment address in the integrated form. Integrated address contains destination address + payment ID. In that case we want to display on Trezor the integrated address as user entered it, otherwise he wouldn't be able to compare it. 

`integrated_indices` contains indices of the txout transfers entered as an integrated address.